src/blockchain/ain/jellyfish/utils/raw-tx-account.ts

```diff
@@ -0,0 +1,71 @@
+import BigNumber from 'bignumber.js';
+import { Config } from 'src/config/config';
+import { RawTxDto } from '../dto/raw-tx.dto';
+import { RawTxBase } from './raw-tx-base';
+import { RawTxUtil } from './raw-tx-util';
+
+export class RawTxAccount extends RawTxBase {
+  async sendFromTo(from: string, to: string, token: number, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() => this.send(from, to, token, amount));
+  }
+
+  async sendToLiq(from: string, token: number, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(
+        from,
+        Config.yieldMachine.liquidity.address,
+        token,
+        amount,
+        new BigNumber(Config.payIn.forward.accountToAccountFee),
+      ),
+    );
+  }
```

Comment:
Same here ... I wouldn't tie it to the liquidity address but search for the real logic. In this case it's the selection of the fee UTXO.